src/cript/api/utils/save_helper.py

```diff
@@ -53,3 +62,18 @@ def find_node_by_uuid(node, uuid: str):
     missing_node = node.find_children({"uuid": uuid})[0]
 
     return missing_node
+
+
+def _identify_suppress_attributes(node, response: Dict) -> Optional[Dict[str, Set[str]]]:
+    suppress_attributes = {}
+    if response["error"].startswith("Additional properties are not allowed"):
+        # Find all the attributes, that are listed in the error message with regex
+        attributes = set(re.findall(r"'(.*?)'", response["error"]))
+        path = response["error"][response["error"].rfind("path:") + len("path:") :].strip()
+
+        if path != "/":
+            # TODO find the UUID this belongs to
+            raise NotImplementedError("Fixing non-root objects for patch, not implemented yet")
```

Comment:
I suggest raising a generic exception for this

```suggestion
raise Exception("Fixing non-root objects for patch, not implemented yet")
```